docs/map.xsd

```diff
@@ -247,22 +452,33 @@
 </xs:attributeGroup>
 
 <xs:attributeGroup name="tile.data.layer">
-  <xs:attribute name="gid" type="xs:nonNegativeInteger" use="required"/>
+  <xs:attribute name="gid" type="tileId" use="required"/>
 </xs:attributeGroup>
 
 <xs:attributeGroup name="objectgroup">
+  <!--Not sure if missing x,y which are defaulted to 0 and unchangable-->
```

Comment:
In the code, these x/y Layer attributes are an offset in tile units, but this is not supported for object groups by the current version of Tiled. I may rearrange things in libtiled to get rid of them and I think they should just be left out.